### ISSUE TITLE
Fig Ai command returns 429

### ISSUE THREAD

**Issue by xvalentino** [withfig/fig#2897]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 14.2.1 (23C71)

### Expected behaviour

`fig ai`
Text > "test"

Returns some kind of ai generated response.

Fig update doesn't seem to resolve, despite seeing [this](https://github.com/withfig/fig/issues/2842) one error where this bug seems to fixed

### Actual behaviour

```
failed to generate suggestions: Request failed with status code 429 (status: 500 Internal Server Error, error_id: dcf54076c4704b6eaf92f8cee9135be5)
```

### Steps to reproduce

```bash
  fig ai echo hello
```

Also when I run

```
fig restart
```
I get 
```
Something went wrong :(
Error: Missing submessage
```

### Environment

```yaml
fig-details:
  - 2.17.1
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Max
  - cores: 10
  - mem: 32.00 GB
os-info:
  - macOS 14.2.1 (23C71)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/valentino/dev/handshake
  - exe-path: /Users/valentino/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 8d4b2a53-868a-488b-acf3-38f2af6f0b10
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.17.1
    - PATH: /Users/valentino/.bun/bin:/Users/valentino/.rbenv/shims:/Users/valentino/Library/Caches/fnm_multishells/1028_1706656261845/bin:/Users/valentino/Library/Application Support/fnm:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/Users/valentino/.fig/bin:/Users/valentino/.local/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.googlecode.iterm2
    - FIG_PID: 997
    - FIG_SET_PARENT: 8d4b2a53-868a-488b-acf3-38f2af6f0b10
```

**Comment by mschrage** [comment]
Hi @xvalentino! Thanks for reporting, this issue should be resolved now. (We bumped into a spending cap and have increased the limit)

**Comment by duriann** [comment]
the same to me

**Comment by declanchiu** [comment]
me too

**Comment by Turgutalp** [comment]
me too

**Comment by yaronEcoPlant** [comment]
me too

**Comment by aimranov27** [comment]
same for me, issue is not fixed

**Comment by xvalentino** [comment]
Is there anything I need to do to get this fix applied? I still seem to get the error

**Comment by atsneves** [comment]
same for me, issue is not fixed

**Comment by xvalentino** [comment]
<img width="1296" alt="Screenshot 2024-01-31 at 5 51 05 PM" src="https://github.com/withfig/fig/assets/8762410/c1c3f929-f3ab-4b32-8d5e-6ace31fecd90">
Devtools don't give much help

**Comment by randel-reyes22** [comment]
Same with me, any fix?

<img width="1016" alt="image" src="https://github.com/withfig/fig/assets/53185716/72909aeb-b056-43af-a562-eb01ce6fc4ae">

**Comment by vincentvella** [comment]
@mschrage I think this needs to be reopened, doesn't look like the fix you referred to did anything to unblock users

**Comment by xvalentino** [comment]
I've tried restarting, and reinstalling the fig app. With no luck

**Comment by ybentz** [comment]
Same here

**Comment by henry798798** [comment]
bump - same here

**Comment by gguip** [comment]
same problem here

**Comment by billgertz** [comment]
even after installing CodeWisperer the problem persists.

**Comment by iLudolf** [comment]
Same here

**Comment by VitalijOd** [comment]
I've got the same error

**Comment by zhast** [comment]
Me as well!

**Comment by grant0417** [comment]
Sorry about this issue, a fix has been deployed. Open the dashboard and hit `⌘+r` and the new version should load which will resolve the issue.

**Comment by brendanfalk** [comment]
^that comment is related to the "missing submessage" issue some users have reported.

We are investigating the AI issue now. Looks like there is something wrong with our API keys... Will update soon

**Comment by grant0417** [comment]
The `fig ai` issue is now resolved, closing this issue.
